### PR TITLE
Live stream improvements

### PR DESCRIPTION
**Pull request by fantapop**

### CODE REVIEW

**Comment by fantapop** [comment]
Note: some of the commits here were sitting on my box for the last 6 months.  I went ahead and added them here.  The new stuff is all in "infra/scripts to support live-video"

**Comment by brian-traceme** [comment]
PR #1 !!!!

File: conf/prod.yaml [hunk 1]
```diff
@@ -6,8 +6,7 @@
     { "ParameterKey": "WebsiteServiceCount", "ParameterValue": "2" },
     { "ParameterKey": "ProductServiceCount", "ParameterValue": "2" },
     { "ParameterKey": "DatabaseAdminUsername", "ParameterValue": "tracemeprodroot" },
-    { "ParameterKey": "DatabaseAdminPassword", "ParameterValue": "d4da6Uf@drUHucEbRuspe7$uVad7XA
-      " },
```
Comment (brian-traceme) [inline conf/prod.yaml:10]: What? the password doesn't have a newline? that would be super secure!

File: stacks/database.yaml [hunk 2]
```diff
@@ -76,8 +76,10 @@ Resources:
       AllocatedStorage: !Ref StorageSizeGigabytes
       DBInstanceClass: !Ref InstanceType
       DBInstanceIdentifier: !Sub ${EnvironmentName}-rds-pg
+
       Engine: postgres
       EngineVersion: 9.6
+      MultiAZ: true
```
Comment (brian-traceme) [inline stacks/database.yaml:82]: multiple arizona?

**Pull request merged**